src/main/java/org/tron/common/runtime/vm/program/Stack.java

```diff
@@ -59,4 +60,25 @@ public void swap(int from, int to) {
   private boolean isAccessible(int from) {
     return from >= 0 && from < size();
   }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Reduce the number of returns of this method 4, down to the maximum allowed 3. ([squid:S1142](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1142))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1681&issues=AWbnEHQ6SgN4uCcljsr3&open=AWbnEHQ6SgN4uCcljsr3)

[ISSUE_KEY]: AWbnEHQ6SgN4uCcljsr3